src/device/mod.rs

```diff
@@ -201,7 +201,7 @@ pub struct RequestedState {
     // Requested keepalive periods
     pub(crate) keepalive_periods: FeaturePersistentKeepalive,
 
-    pub postquantum_wg: Option<wg::pq::PqKeys>,
+    pub postquantum_keys: Option<telio_pq::Keys>,
```

Comment:
While not a strict rule or anything, the `requested_state` struct generally represents configuration requests coming from outside of libtelio. And internally derived/calculated/communicated things are stored directly in the entities themselves, which may be made public by the choice of said component.

We do have exception, like `wg_stun_server`, but it is generally too small to contain its own active component, and thus got here, even though does not really fit the purpose of the struct.

But in `telio-pq` case, it is already active component, therefore maybe some getters would make it fit better?

One example of such getter in a rather small active component is [upgrade_sync](https://github.com/NordSecurity/libtelio/blob/main/crates/telio-traversal/src/upgrade_sync.rs), and the getter is defined [here](https://github.com/NordSecurity/libtelio/blob/main/crates/telio-traversal/src/upgrade_sync.rs#L95), and example call site is [here](https://github.com/NordSecurity/libtelio/blob/main/src/device/wg_controller.rs#L553)

@matszczygiel what do you think about moving keys to the telio-pq crate itself instead of adding them into requested state?

Reply:
I agree about moving keys from `RequestedState` however I don't like moving it to telio-pq because the key handling logic is (and must be) implemented in `device/mod.rs`. Thus even if we move keys storage there, they still need to be set up from the device making a weird dependency loop.
I think I will try to move them into the entity stored in the device but it would be a different entity than now, something like
```rust
struct PqEntity {
    keys: telio_pq::Keys,
    conn: telio_pq::Entity, // I'll probably change the type name in this case
}
```

Reply:
Done as described, but moved the entity nonetheless into `telio-pq` crate